src/js/index.js

```diff
@@ -13,6 +13,74 @@ const navBookmark = document.querySelector('[data-js=navBookmark]')
 const navCreate = document.querySelector('[data-js=navCreate]')
 const navSettings = document.querySelector('[data-js=navSettings]')
 
+// Get main sections (^= pages)
+const pages = document.querySelectorAll('.page')
+
+// Get each main section (^= page)
+const createPage = document.querySelector('.create-page')
+const homePage = document.querySelector('.app-grid')
+
+// Get Header Heading
+const headerHeading = document.querySelector('h1')
+
+// Get form
+const form = document.querySelector('form')
+
+// Get textareas
+const questionArea = document.querySelector('.create-page__form__add-question')
+const answerArea = document.querySelector('.create-page__form__add-answer')
+const tagsArea = document.querySelector('.create-page__form__add-tags')
+
+// Get character counters
+const questionCounter = document.querySelector(
+  '.create-page__form__add-question__counter'
+)
+const answerCounter = document.querySelector(
+  '.create-page__form__add-answer__counter'
+)
+const tagCounter = document.querySelector(
+  '.create-page__form__add-tags__counter'
+)
+
+// Get button
+const formButton = document.querySelector('.create-page__form__button')
+
+/* ALPHA TESTING 
+tagsArea.addEventListener('input', () => {
+  return tagCounter.textContent.match(/,/g)
+  ? (tagCounter.textContent = tagCounter.textContent.match(/,/g).length - 5)
+  : (tagCounter.textContent = 5)
+})*/
+
+/* Functions */
+
+// Prevent form from submitting & delete all input
+form.addEventListener('keyup', function (event) {
+  if (event.key === 'Enter') {
+    formButton.click()
+  }
+})
+
+form.addEventListener('submit', event => {
+  event.preventDefault()
+
+  // add lines that return form input and make new card
+
+  form.reset()
+  questionArea.focus()
+})
+
+// Add counters to textareas
+
+questionArea.addEventListener('input', () => {
+  questionCounter.textContent =
+    questionArea.maxLength - questionArea.value.length
+})
+
+answerArea.addEventListener('input', () => {
```

Comment:
You could use the same logic for counting the question and answer lengths. Just like we did with multiple cards and their answer button logic.

Reply:
Wow, that's an ambitious thing to try (at least with what we learned so far). I like that you tried it. Let's have a talk about that tomorrow with the other students. Please remind me!